### PR TITLE
Mstp import

### PR DESCRIPTION
**Pull request by orangewolf**

### CODE REVIEW

File: lib/import_old.rb [hunk 1]
```diff
@@ -1,27 +1,191 @@
...
+
+
+    # map the rest of the academic record fields
+
+
+    # complete = applicant.validates_application_completeness
+    # if !complete
+    #   raise "Application not complete because:\n #{applicant.errors.full_messages.join("\n")}"
+    # end
+    binding.pry
```
Comment (orangewolf) [inline lib/import_old.rb]: this needs to be removed

File: lib/import_old.rb [hunk 2]
```diff
@@ -1,27 +1,191 @@
...
-#    next unless user.academic_record.present?
+# user = Old::User.find 257
 
-    applicant = Applicant.new
-    applicant.first_name = user.firstname
-    # map the rest of the user fieds
-
-    academic_record = applicant.records.build
+Old::User.all.each do |user|
+ # begin
```
Comment (orangewolf) [inline lib/import_old.rb]: this and the rescue at the bottom need to be uncommented

File: lib/import_old.rb [hunk 3]
```diff
@@ -1,27 +1,191 @@
...
+        applicant.other_funds = old_extra.personal_statement,
+        applicant.found_us = old_extra.learn
+        applicant.gpa_comment = old_extra.gpa_comments,
+        applicant.additional_info = old_extra.additional_info
     end
 
-    applicant.save!
+    # applicant.first_name = user.firstname
+    # applicant.last_name = user.lastname
+    # applicant.phone = user.phone
```
Comment (orangewolf) [inline lib/import_old.rb]: all off this commented code needs to be removed if those fields are all accounted for

File: lib/import_old.rb [hunk 4]
```diff
@@ -1,27 +1,191 @@
...
+
+
+    # user.recommendations.each do |r|
+    #     recommendation = applicant.recommendations.build(
+    #         known_applicant_for: r.known_student,
+    #         known_capacity: r.know_capacity,
+    #         overall_promise: r.rating,
+    #         undergraduate_institution: r.undergrad_inst,
+    #         body: r.faculty_comment,
+    #         applicant_id: r.user_id,
```
Comment (orangewolf) [inline lib/import_old.rb]: are these fields all covered below? If so can we clean this up?

File: lib/import_old.rb [hunk 5]
```diff
@@ -1,27 +1,191 @@
...
+    # academic_record = applicant.records.build
+    # academic_record.university = old_academic_record.college
+    # academic_record.applicant_id = old_academic_record.user_id
+    # applicant.lab_skills = old_extra.lab_skills
+    # applicant.cpu_skills = old_extra.comp_skills
+    # applicant.academic_level = old_academic_record.college_level
+    # applicant.statement = old_extra.personal_statement
+    # academic_record.start = old_academic_record.college_start
+    # academic_record.finish = old_academic_record.college_end
+    # academic_record.gpa = old_academic_record.gpa
```
Comment (orangewolf) [inline lib/import_old.rb]: you know what I'm going to ask here ;-)

File: lib/import_old.rb (hunk 2 above)
Comment (donohue76) [inline lib/import_old.rb]: this was completed

File: lib/import_old.rb (hunk 3 above)
Comment (donohue76) [inline lib/import_old.rb]: commented code has been removed

File: lib/import_old.rb (hunk 4 above)
Comment (donohue76) [inline lib/import_old.rb]: commented code has been removed

File: lib/import_old.rb [hunk 6]
```diff
@@ -1,27 +1,191 @@
...
+                applicant_id: old_recommendation.user_id,
+                recommender_id: old_recommendation.recommender_id,
+                created_at: old_recommendation.created_at,
+                updated_at: old_recommendation.updated_at
+            )
+        else
+            recommendation = applicant.recommendations.build()
+        end
+        recommendation.recommender = Recommender.new(
+            first_name: r.name,
```
Comment (donohue76) [inline lib/import_old.rb]: this format has been implemented

File: lib/import_old.rb (hunk 5 above)
Comment (donohue76) [inline lib/import_old.rb]: this code has been accounted for and removed

File: lib/import_old.rb (hunk 1 above)
Comment (donohue76) [inline lib/import_old.rb]: code accounted for and removed

**Pull request merged**